parts/k8s/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -73,6 +73,32 @@ NVIDIA_DOCKER_VERSION=2.0.3
 DOCKER_VERSION=1.13.1-1
 NVIDIA_CONTAINER_RUNTIME_VERSION=2.0.0
 
+configure_prerequisites() {
+    if which apt > /dev/null; then
+        # Install required packages; only for apt-based systems
+        apt update -qq
+        apt install -y curl gpg jq nfs-common
+    fi
+
+    modprobe br_netfilter overlay
```

Comment:
Can this be integrated into the existing `configureCNI` func? (in `cse_config.sh`)

Reply:
Looks like br_netfilter is already there. It's possible overlay isn't even needed. Will run a test...

Reply:
I've removed these - br_netfilter is already there and I see no difference with overlay missing.